site/src/docs/foundation/design-tokens/breakpoints/index.mdx

```diff
@@ -4,28 +4,32 @@ title: 'Breakpoint tokens - Usage'
 navTitle: 'Breakpoints'
 ---
 
-import ExternalLink from '../../../../components/ExternalLink'
+import ExternalLink from '../../../../components/ExternalLink';
 import TabsLayout from './tabs.mdx';
 
 export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
 
 ## Usage
-HDS includes tokenized values for both breakpoints and a maximum container width. Breakpoint tokens can be also used alongside <Link size="M" href="#">HDS grid guidelines</Link> in order to create consistent designs and implementations.
+
+HDS includes tokenized values for both breakpoints and a maximum container width. Breakpoint tokens can be also used alongside <Link size="M" href="/foundation/guidelines/grid">HDS grid guidelines</Link> in order to create consistent designs and implementations.
```

Comment:
There might be couple ways to improve this link:

1. Add gatsby's withPrefix function here. Currently the link is not working in the demo environment. The withPrefix should fix that.
2. Use dynamic link routing instead of href. The benefit is, that then there is no page reload. There might be 2 ways to achieve this, but atleast something like this should work. Not sure if it needs the withPrefix at all if you use this method:

```
onClick={(event) => {
 event.preventDefault();
 navigate('/foundation/guidelines/grid')
}}
```

Import navigate from gatsby.

Reply:
Plaah, I forgot these. We need to the prefix to make the links works in the demo as mentioned earlier

Reply:
Okay, I think I got everything. Made InternalLink component, it uses withPrefix and changed every internal link to use it except a couple of same page links. Go ahead and review.